### PR TITLE
Move EIP-4626 to Final 🎉

### PR DESCRIPTION
**Pull request by Joeysantoro**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by alcueca** [comment]
Ready to move to Final on my side.

**Review by fubuloubu — APPROVED** [review]

File: EIPS/eip-4626.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Tokenized Vault Standard
 description: A standard for tokenized Vaults with a single underlying ERC-20 token.
 author: Joey Santoro (@joeysantoro), t11s (@transmissions11), Jet Jadeja (@JetJadeja), Alberto Cuesta Cañada (@alcueca), Señor Doggo (@fubuloubu)
 discussions-to: https://ethereum-magicians.org/t/eip-4626-yield-bearing-vault-standard/7900
-status: Last Call
+status: Final
 last-call-deadline: 2022-03-11
```
Comment (lightclient) [inline EIPS/eip-4626.md:8]:
```suggestion
```

**Review by lightclient — CHANGES_REQUESTED** [review]

File: EIPS/eip-4626.md (hunk 1 above)
Comment (lightclient) [reply EIPS/eip-4626.md:8]: For some reason it’s not letting me apply this suggestion.

Comment (Joeysantoro) [reply EIPS/eip-4626.md:8]: done!

**Review by lightclient — APPROVED** [review]
LGTM

**Review by azboyzranch — COMMENTED** [review]
>.

**Comment by azboyzranch** [comment]
> LGTM

**Pull request merged**